ntfysh_client/Notifications/NotificationListener.cs

```diff
@@ -350,5 +351,11 @@ public async Task UnsubscribeFromTopicAsync(string topicUniqueString)
             //Remove the old topic
             SubscribedTopicsByUnique.Remove(topicUniqueString);
         }
+
+        public async Task<HttpResponseMessage> SendNotification(string host, NtfyEvent message)
+        {
+            var httpClient = new HttpClient();
+            return await httpClient.PostAsJsonAsync<NtfyEvent>(host, message);
```

Comment:
This doesn't appear to handle credentials on a topic. You can retrieve the topic from `SubscribedTopicsByUnique` within this class